server/util.go

```diff
@@ -70,9 +70,6 @@ func PrintConfigCheckMsg(cfg *config.Config) {
 // CheckPDVersion checks if PD needs to be upgraded.
 func CheckPDVersion(opt *config.PersistOptions) {
 	pdVersion := versioninfo.MinSupportedVersion(versioninfo.Base)
-	if versioninfo.PDReleaseVersion != "None" {
-		pdVersion = versioninfo.MustParseVersion(versioninfo.PDReleaseVersion)
-	}
```

Comment:
why remove this?

Reply:
This is always true.

Reply:
Not really. It can be set to another value by makefile https://github.com/tikv/pd/blob/4d4d293bcb9ea393506a3cc5b7872239c2a8324d/Makefile#L62

Reply:
Fixed.